### ISSUE TITLE
[CRITICAL] brandable_css/bin/brandable_css fails with new version of npm node-sass in Canvas LMS

### ISSUE THREAD

**Issue by Kulgar** [instructure/canvas-lms#811]
### Summary:

Breaking change in node-sass 3.5.3 (although they don't say it is a breaking change: https://github.com/sass/node-sass/issues/1471#issuecomment-213404643 )

brandable_css/bin/brandable_css no longer works because of that. 

### Steps to reproduce:

 1. Update your npm modules 
 2. Check that it did install node-sass 3.5.3
 3. Try to run ./node_modules/brandable_css/bin/brandable_css

### Expected behavior:

You should see this error: 

```bash
error compiling sass { [Error: Import directives may not be used within control directives or mixins.]
  cause: 
   { [Error: Import directives may not be used within control directives or mixins.]
     formatted: 'Error: Import directives may not be used within control directives or mixins.\n        on line 32 of app/stylesheets/base/_variables.scss\n>>   @import "brand_variables";\n   --^\n',
     message: 'Import directives may not be used within control directives or mixins.',
     column: 3,
     line: 32,
     file: 'app/stylesheets/base/_variables.scss',
     status: 1 },
  isOperational: true,
  formatted: 'Error: Import directives may not be used within control directives or mixins.\n        on line 32 of app/stylesheets/base/_variables.scss\n>>   @import "brand_variables";\n   --^\n',
  column: 3,
  line: 32,
  file: 'app/stylesheets/base/_variables.scss',
  status: 1 } Error: Import directives may not be used within control directives or mixins.
```

### Additional notes:

This is not the only breaking change that prevents brandable_css from properly running. 

I posted all the details here: 
https://github.com/ryankshaw/brandable_css/issues/3

You may also take a look at: https://github.com/sass/node-sass/issues/1471

**Comment by claydiffrient** [comment]
@Kulgar I believe the fix (9dfe5bf055127aa22b5cf15b362bb65ec3fc80b6) is currently on the master branch and will likely be picked over to the stable branch soon.

**Comment by Kulgar** [comment]
The fix is related to 3.5.1. It seems to fix the @import bug but not sure if it fixes the second one: 

```bash
error compiling sass { [Error: Undefined variable: "$lightBackground".]
  cause: 
   { [Error: Undefined variable: "$lightBackground".]
     formatted: 'Error: Undefined variable: "$lightBackground".\n        on line 5 of app/stylesheets/bundles/account_admin_tools.scss\n>>     background-color: $lightBackground;\n   ----------------------^\n',
     message: 'Undefined variable: "$lightBackground".',
     column: 23,
     line: 5,
     file: '/home/regmillet/Workspace/RailsProjects/canvas-lms/app/stylesheets/bundles/account_admin_tools.scss',
     status: 1 },
  isOperational: true,
  formatted: 'Error: Undefined variable: "$lightBackground".\n        on line 5 of app/stylesheets/bundles/account_admin_tools.scss\n>>     background-color: $lightBackground;\n   ----------------------^\n',
  column: 23,
  line: 5,
  file: '/home/regmillet/Workspace/RailsProjects/canvas-lms/app/stylesheets/bundles/account_admin_tools.scss',
  status: 1 } Error: Undefined variable: "$lightBackground".
    at options.error (/home/regmillet/Workspace/RailsProjects/canvas-lms/node_modules/node-sass/lib/index.js:271:32) Undefined variable: "$lightBackground".
```

This one occurs when I remove the @import line with node-sass 3.5.3.

**Comment by Kulgar** [comment]
Sorry, just noticed that he forced the version to 3.4.2 :)
